File: src/Codility/Lessons/4-1_frogRiverOne.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(X, A):
    # write your code in Python 3.8.10
    s = set()
    time = -1
    for i in range(1, X + 1):
        s.add(i)
    while s:
        time += 1
        el = A[0]
        if el in s:
            s.remove(el)
        A.pop(0)
    return time

File: src/Codility/Lessons/4-1_frogRiverOne.cpp
#include <iostream>
#include <vector>

using namespace std;
// you can use includes, for example:
// #include <algorithm>

// you can write to stdout for debugging purposes, e.g.
// cout << "this is a debug message" << endl;

int solution(int X, vector<int> &A) {
    // write your code in C++14 (g++ 6.2.0)
    int max = 0;
    for (int n: A)
    {
        if (n > max)
        {
            max = n;
        }
    }
    return max;
}
